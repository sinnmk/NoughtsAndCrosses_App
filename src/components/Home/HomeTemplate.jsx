import React from 'react';

const HomeTemplate = () => {
    return (
        <div>Home</div>
    );
}

export default HomeTemplate;