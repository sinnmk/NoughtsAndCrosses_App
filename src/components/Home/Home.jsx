import { Component } from 'react';
import 'reactstrap'
import HomeTemplate from './HomeTemplate';

export default class Home extends Component {
    constructor(props){
        super(props);
        this.state = {}
    }

    render() {
        return HomeTemplate.call(this);
    }
}