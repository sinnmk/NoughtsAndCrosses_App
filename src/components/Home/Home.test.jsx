import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home'

describe('When Home is rendered', () =>{
    test('Home content is rendered', () =>{
        const component = renderer.create(
            <Home/>).toJSON();

        expect(component).toMatchSnapshot();
    })
})