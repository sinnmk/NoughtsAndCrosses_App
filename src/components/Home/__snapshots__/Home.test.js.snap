// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Home is rendered Home content is rendered 1`] = `
<div>
  Home
</div>
`;
